website/content/partials/components/platform-aws-lambda.mdx

```diff
@@ -0,0 +1,85 @@
+## aws-lambda (platform)
+
+Deploy functions as OCI Images to AWS Lambda.
+
+### Interface
+
+- Input: **ecr.Image**
+- Output: **lambda.Deployment**
+
+### Examples
+
+```hcl
+deploy {
+	use "aws-lambda" {
+		region = "us-east-1"
+		memory = 512
+	}
+}
+```
+
+### Required Parameters
+
+These parameters are used in the [`use` stanza](/docs/waypoint-hcl/use) for this plugin.
+
+#### region
+
+The AWS region for the ECS cluster.
+
+- Type: **string**
+
+### Optional Parameters
+
+These parameters are used in the [`use` stanza](/docs/waypoint-hcl/use) for this plugin.
+
+#### iam_role
+
+An IAM Role specified by ARN that will be used by the Lambda at execution time.
+
+- Type: **string**
+- **Optional**
+- Default: created automatically
+
+#### memory
+
+The amonut of memory, in megabytes, to assign the function.
```

Comment:
I know these are generated so you'll have to edit the source and not directly commit this, but FYI
```suggestion
The amount of memory, in megabytes, to assign the function.
```